themes/docsy/assets/js/base.js

```diff
@@ -27,13 +27,10 @@ limitations under the License.
 
     //Samples filters
     $(".filters .filter").click(function () {
-      var filter = $(this).attr("filter");
-      $(this).addClass("active").siblings().removeClass("active");
-      $(".filter-item").hide();
-      $(".filter-item[class*='" + filter + "']").show();
-      if (filter == "all") {
-        $(".filter-item").show();
-      }
+      if (this.classList.contains("active")) {
+        this.classList.remove("active");
+      } else this.classList.add("active");
```

Comment:
John, friends don't let friends use braces for half of an if statement and omit them on the other half :-)

Reply:
Fixed.